src/js/components/Button/stories/Busy.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import { Box, Button } from 'grommet';
+
+export const Busy = () => {
+  const [busy, setBusy] = useState();
+  const [success, setSuccess] = useState();
+
+  return (
+    <Box align="center" pad="medium">
+      <Button
+        primary
+        busy={busy}
+        success={success}
+        label="Button Busy"
+        onClick={() => {
+          if (success) {
+            setSuccess(false);
+          } else {
+            setBusy(true);
+            setTimeout(() => {
```

Comment:
Do we need a clearTimeout somewhere?

Reply:
It sounds like when you use `setInterval` you need to clear it but if you just use `setTimeout` it is not necessary to clear it. Related stackoverflow thread: https://stackoverflow.com/questions/7391567/when-using-settimeout-do-you-have-to-cleartimeout

Reply:
Great, thanks for clarifying!